assets/stylesheets/kitten/utilities/spacing/_margin.scss

```diff
@@ -42,6 +46,10 @@
     margin-top: k-px-to-rem(30px);
   }
 
+  .k-u-margin-top-quadruple {
+    margin-top: k-px-to-rem(40px);
```

Comment:
Je trouve que `k-px-to-rem($base * 4);` sera plus simple et lisible pour ce cas d'usage.